.github/workflows/conda.yml

```diff
@@ -1,32 +1,182 @@
 ---
-name: Build conda packages
+name: Conda Packages
 on:
   workflow_dispatch:
+    inputs:
+      release-channel:
+        type: choice
+        default: khiops-dev
+        options: [khiops-dev, khiops]
+        description: Anaconda channel to release
+  push:
+    tags: ['*']
+  pull_request:
+    paths:
+      - packaging/conda/**
+      - '!packaging/conda/README.md'
+      - .github/workflows/conda.yml
+defaults:
+  run:
+    shell: bash -el {0}
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref
+    }}
+  cancel-in-progress: true
 jobs:
-  conda-build:
+  build:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-2022, macos-latest]
+        include:
+          - os: ubuntu-22.04
+            os-family: linux
+          - os: windows-2022
+            os-family: windows
+          - os: macos-13
+            os-family: macos
     runs-on: ${{ matrix.os }}
     steps:
-      - name: Checkout sources
-        uses: actions/checkout@v3
-      - name: Install miniconda
-        uses: conda-incubator/setup-miniconda@v2
-        with:
-          python-version: '3.10'
-          auto-activate-base: true
-          activate-environment: true
-      - name: Install conda-build
-        run: conda install conda-build
-      - name: Build khiops-bin conda package
+      - name: Checkout Sources
+        uses: actions/checkout@v4
+      - name: Install Miniconda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          miniconda-version: latest
+          python-version: '3.11'
+      - name: Install Dependency Requirements for Building Conda Packages
+        run: conda install conda-build conda-verify
+      - name: Put the Khiops version in the environment
+        shell: bash
+        run: |
+          if [[ "${{ github.ref_type }}" == "tag" ]]
+          then
+            echo "KHIOPS_VERSION=${{ github.ref_name }}" >> "$GITHUB_ENV"
+          else
+            echo "KHIOPS_VERSION=$(./scripts/khiops-version)" >> "$GITHUB_ENV"
+          fi
```

Comment:
C'est pas homogène avec les autres paquets windows, deb ou rpm si ?

Reply:
More or less. It is coherent with the others in the sense that the tag overrides the version in the sources (which is the idea of `./khiops-package-version`) but we don't add `preview` or anything.

Maybe we should discuss #100 soon ?

Reply:
To be resolved when we fix #100